meeteval/wer/wer/mimo.py

```diff
@@ -107,10 +67,60 @@ def get_keys(obj):
     )
 
 
+def mimo_word_error_rate(
+        reference: 'List[List[str] | Dict[Any, str]] | Dict[List[str], Dict[Any, str]] | STM',
+        hypothesis: 'List[str] | Dict[str] | STM',
+) -> MimoErrorRate:
+    """
+    The Multiple Input speaker, Multiple Output channel (MIMO) WER.
+
+    >>> mimo_word_error_rate([['a b c d e f']], ['a b c d e f'])
+    MimoErrorRate(errors=0, length=6, insertions=0, deletions=0, substitutions=0, error_rate=0.0, assignment=[(0, 0)])
+
+    # All correct, utterance order between speakers can change
+    >>> mimo_word_error_rate([['a b', 'c d'], ['e f']], ['a b', 'e f c d'])
+    MimoErrorRate(errors=0, length=6, insertions=0, deletions=0, substitutions=0, error_rate=0.0, assignment=[(1, 1), (0, 0), (0, 1)])
+    >>> mimo_word_error_rate([['a b', 'c d'], ['e f']], ['a b', 'c d e f'])
+    MimoErrorRate(errors=0, length=6, insertions=0, deletions=0, substitutions=0, error_rate=0.0, assignment=[(0, 0), (0, 1), (1, 1)])
+    >>> mimo_word_error_rate([['a b', 'c d'], ['e f']], ['c d', 'a b e f'])
+    MimoErrorRate(errors=0, length=6, insertions=0, deletions=0, substitutions=0, error_rate=0.0, assignment=[(0, 1), (1, 1), (0, 0)])
+
+    >>> mimo_word_error_rate({'A': ['a b', 'c d'], 'B': ['e f']},
+    ...                      {'O1': 'c d', 'O2': 'a b e f'})
+    MimoErrorRate(errors=0, length=6, insertions=0, deletions=0, substitutions=0, error_rate=0.0, assignment=[('A', 'O2'), ('B', 'O2'), ('A', 'O1')])
+
+    """
+    if isinstance(reference, STM) or isinstance(hypothesis, STM):
+        assert isinstance(hypothesis, STM) and isinstance(reference, STM)
```

Comment:
`assert isinstance(hypothesis, STM) and isinstance(reference, STM), (type(hypothesis), type(reference))`